### DISCUSSION TITLE
disk fragmentation issues

### DISCUSSION THREAD

**Discussion by zephyr26531** [seaweedfs/seaweedfs#5329] [Q&A]
I wanna ask that how seaweedfs handles disk fragmentation issues?

**Comment by SmsS4** [comment]
I don't think seaweedfs handles disk fragmentation :-? (At least I didn't see anything about it)
If I'm not wrong you can use `volumePreallocate` in Master and I think it will reduce disk fragmentation :-? (Not sure)
https://github.com/seaweedfs/seaweedfs/wiki/Optimization#preallocate-volume-file-disk-spaces

**Comment by chrislusf** [comment]
SeaweedFS on disk data files are append only. There are no disk fragmentation.
